surveys/templates/survey_submitted_detail.html

```diff
@@ -38,7 +38,7 @@ <h2 class="card-title">{{ form_entry.name }}</h2>
 <script>
 $(document).ready(function () {
   $('#survey-submitted-detail').DataTable({
-    "scrollX": true
+    // "scrollX": true,
```

Comment:
What was going on here?

Reply:
DataTables' ScrollX styling was wonky certain kinds of submitted data. Got it working with this fix! https://stackoverflow.com/questions/45136218/datatables-scrollx-causing-squashed-header?rq=1

Reply:
Nice! Do you mind adding a quick comment to this code describing the problem and linking to this SO answer? (You can use the "share" URL for brevity: https://stackoverflow.com/a/47800221) I know in 6 months I'm not going to remember why we added these options.